micrometer-core/src/main/java/io/micrometer/core/instrument/distribution/DistributionStatisticConfig.java

```diff
@@ -271,7 +291,15 @@ public double[] getServiceLevelObjectiveBoundaries() {
 
     public static class Builder {
 
-        private final DistributionStatisticConfig config = new DistributionStatisticConfig();
+        protected final DistributionStatisticConfig config;
+
+        public Builder() {
+            config = new DistributionStatisticConfig();
+        }
+
+        protected Builder(DistributionStatisticConfig config) {
```

Comment:
I'm not sure I get this either a `Builder` should create an instance of `DistributionStatisticConfig`, not receive one.